### PR TITLE
feat: add a bunch of dark sites

### PR DESCRIPTION
**Pull request by pizzafox**
Adds a bunch of dark sites I've been collecting in my personal site list over the years.

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -99,6 +99,7 @@ codepen.io
 codesandbox.io
 codingame.com
 coinpot.co
+colab.research.google.com
```
Comment (Gusted) [inline src/config/dark-sites.config:102]: Not dark by default

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -111,9 +112,12 @@ cracked.to
 crackingking.com
 crackwatch.com
 crazygames.com
+create-react-app.dev
```
Comment (Gusted) [inline src/config/dark-sites.config:115]: Not dark by default

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -111,9 +112,12 @@ cracked.to
...
 cryptowat.ch
 cs.rin.ru
 cswarzone.com
+ctr-electronics.com
+cutestrap.com
```
Comment (Gusted) [inline src/config/dark-sites.config:120]: Not dark by default

File: src/config/dark-sites.config [hunk 4]
```diff
@@ -153,6 +157,7 @@ discord.com/developers
 discord.com/invite
 discord.com/login
 discord.com/register
+discord.js.org
```
Comment (Gusted) [inline src/config/dark-sites.config:160]: Not dark by default

File: src/config/dark-sites.config [hunk 5]
```diff
@@ -165,6 +170,7 @@ dlive.tv
 docs.docker.com
 dodi-repacks.site
 donationalerts.com
+dormdev.com
```
Comment (Gusted) [inline src/config/dark-sites.config:173]: Not dark by default

File: src/config/dark-sites.config [hunk 6]
```diff
@@ -174,8 +180,10 @@ draculatheme.com
 draftkings.com
 drakewars.com
 drunkenslug.com
+duckduckgo.com
```
Comment (Gusted) [inline src/config/dark-sites.config:183]: Not dark by default

File: src/config/dark-sites.config [hunk 7]
```diff
@@ -326,6 +336,7 @@ kissmanga.com
 kittybot.de
 kiwifarms.cc
 kiwifarms.net
+klasa.js.org
```
Comment (Gusted) [inline src/config/dark-sites.config:339]: Not dark by default

File: src/config/dark-sites.config [hunk 8]
```diff
@@ -347,23 +359,29 @@ m4rtyr.github.io
 macbb.org
 magnetoo.io
 magoware.tv
+mango.pdf.zone
+maroon.jonah.pw
 marte.dev
+material.angular.io
```
Comment (Gusted) [inline src/config/dark-sites.config:365]: Not dark by default

File: src/config/dark-sites.config [hunk 9]
```diff
@@ -347,23 +359,29 @@ m4rtyr.github.io
...
 matteotiscia.com
 mcrpw.github.io
 mcskinhistory.com
 mednafen.github.io
 melody.ml
 merklex.io
+messages.google.com
```
Comment (Gusted) [inline src/config/dark-sites.config:372]: Not dark by default

File: src/config/dark-sites.config [hunk 10]
```diff
@@ -347,23 +359,29 @@ m4rtyr.github.io
...
 metronom.us
 metropool.nl
 miaou.drycat.fr
+mikro-orm.io
```
Comment (Gusted) [inline src/config/dark-sites.config:376]: Not dark by default

File: src/config/dark-sites.config [hunk 11]
```diff
@@ -415,12 +434,15 @@ openemu.org
...
 ovagames.com
+overcoder.dev
 overdodactyl.github.io/ShadowFox
+oxy.cannot.date
 p4pirate.xyz
 parahumans.wordpress.com
 paranoid.email
 pathof.info
 pathofexile.com
+pcpartpicker.com
```
Comment (Gusted) [inline src/config/dark-sites.config:445]: Not dark by default

File: src/config/dark-sites.config [hunk 12]
```diff
@@ -596,6 +625,9 @@ tuner.ninja
 tusharsadhwani.dev
 twelvesmith.com
 twist.moe
+twitch.tv
```
Comment (Gusted) [inline src/config/dark-sites.config:628]: Dark detected by system colors.

File: src/config/dark-sites.config [hunk 13]
```diff
@@ -596,6 +625,9 @@ tuner.ninja
...
+twitter.com
```
Comment (Gusted) [inline src/config/dark-sites.config:629]: Dark detected by system colors.

File: src/config/dark-sites.config [hunk 14]
```diff
@@ -596,6 +625,9 @@ tuner.ninja
...
+typescriptlang.org/docs
```
Comment (Gusted) [inline src/config/dark-sites.config:630]: Not dark by default

File: src/config/dark-sites.config [hunk 15]
```diff
@@ -605,9 +637,11 @@ ussr.obys.agency
 uwatchfree.ax
 v3rmillion.net
 v7player.wostreaming.net
+v8.dev
 vakhtangov.ru
 vanced.app
 vcjhwebdev.github.io/useless-translator/
+vercel.com
```
Comment (Gusted) [inline src/config/dark-sites.config:644]: Not dark by default

**Review by Gusted — CHANGES_REQUESTED** [review]

**Comment by pizzafox** [comment]
@Gusted what's the qualifier for sites marked as "dark detected by system colors", and why is this a bad thing?

**Comment by Gusted** [comment]
> @Gusted what's the qualifier for sites marked as "dark detected by system colors", and why is this a bad thing?

Dark by default no matter what.

That means if the site is checking if the user preferred dark colors. Than if the user has preferred light colors the site will have a light theme and dark reader will say it's in the dark config and not darken it. It's not dark by default

**Pull request merged**